### PR TITLE
Add vrajdesai78 profile

### PR DESCRIPTION
**Pull request by vrajdesai78**
Add vrajdesai78.json file 

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/336"></a>

### CODE REVIEW

**Review by aayush89890 — APPROVED** [review]
Looks good, Welcome to the community @vrajdesai78

**Comment by vrajdesai78** [comment]
> Looks good, Welcome to the community @vrajdesai78

Thank you @aayush89890

File: public/data/vrajdesai78.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        "url": "https://github.com/vrajdesai78",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/vrajdesai78",
+        "icon": "twitter"
+      }
+    ]
+  }
```
Comment (Panquesito7) [inline public/data/vrajdesai78.json:17]: Missing an endline.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Comment by vrajdesai78** [comment]
Why my profile is not there on linkfree homepage ?

**Comment by vrajdesai78** [comment]
Why my profile is not there on linkfree homepage ?

**Comment by vrajdesai78** [comment]
Why my profile is not there on linkfree homepage ?

**Comment by vrajdesai78** [comment]
Why my profile is not there on linkfree homepage ?

**Comment by Panquesito7** [comment]
Please do not spam the same comment. Your profile won't appear immediately; it may take some minutes or even more time for your profile to appear.

**Comment by vrajdesai78** [comment]
> Please do not spam the same comment. Your profile won't appear immediately; it may take some minutes or even more time for your profile to appear.

Sorry I didn't know about that. I won't comment again.

**Pull request merged**